backend/types.ts

```diff
@@ -22,6 +22,17 @@ export type RegisterUserDTO = Omit<CreateUserDTO, "role">;
 
 export type AuthDTO = Token & UserDTO;
 
+export type LessonDTO = {
+  id: string;
+  course: string;
+  title: string;
+  description: string;
+  image: string;
+  content: [Object];
+}
+
+export type CreateLessonDTO = Omit<LessonDTO, "id">;
```

Comment:
For consistency, this should probably be `CreateLessonRequestDTO`, given that it's a request, not an entity.